vms/proposervm/block/parse_test.go

```diff
@@ -44,7 +44,7 @@ func TestParse(t *testing.T) {
 
 	builtBlockBytes := builtBlock.Bytes()
 
-	parsedBlockIntf, err := Parse(builtBlockBytes)
+	parsedBlockIntf, err := Parse(builtBlockBytes, timestamp)
```

Comment:
nit: should we use `version.DefaultUpgradeTime` instead of timestamp = time.Unix(123,0) to signal an preDurango time in this UT?

Reply:
I don't think it really matters tbh